### PR TITLE
Removed IRC from community links/footer

### LINKED ISSUE #605: Remove IRC
Remove IRC from https://www.decred.org/community/ as well as footer (#600).

### PR DESCRIPTION
**Pull request by peterzen**
Closes #605

### CODE REVIEW

File: src/data/community/channels.yml [hunk 1]
```diff
@@ -10,10 +10,6 @@
   url: "https://slack.decred.org"
   img_src: "dcr-community-platforms-slack.png"
   name: Slack
--
-  url: "https://webchat.freenode.net/?channels=decred&uio=d4"
-  img_src: "dcr-community-platforms-irc.png"
```
Comment (jholdstock) [inline src/data/community/channels.yml]: We should be able to remove `dcr-community-platforms-irc.png` as well as any other IRC related images.

Some of them are still referenced in `.css` files, so they may need a review also.

Comment (peterzen) [reply src/data/community/channels.yml]: Ah right, so the idea is to kill it altogether - I'll do a proper cleanup then.

Comment (jholdstock) [reply src/data/community/channels.yml]: I believe that is correct yeah - @jzbz could you confirm?

Comment (jzbz) [reply src/data/community/channels.yml]: Correct. IRC gets nuked.

Comment (peterzen) [reply src/data/community/channels.yml]: Removed all references

Comment (jholdstock) [reply src/data/community/channels.yml]: Link is still appearing in the nav menu (although the icon has been deleted)

Comment (peterzen) [reply src/data/community/channels.yml]: Thanks for spotting, it's gone now

**Pull request merged**